test/drenv/options.py

```
@@ -0,0 +1,50 @@
+
+
+def make_formatter(prog):
+    """
+    Formatter factory for argparse's `formatter_class=`. See
+    wrap_description() for how this ties together with the pre-wrapped
+    description.
+    """
+    return argparse.RawDescriptionHelpFormatter(prog, width=terminal_width())
```

Ruff (0.16.1)

[error] Undefined name `argparse`

(F821)